### DISCUSSION TITLE
LCD standby to any touch, such as iddle

### DISCUSSION THREAD

**Discussion by proasnet** [HASwitchPlate/HASPone#216] [Ideas]
Hi, is possible set standby-mode as iddle, for non backlight display at night?
I know, that this function is available for nextion, I am working with this in my another job projects, I can add this to customized hmi file for LCD. But, is better has it as in your source for next update in a future and setting via HA automation, for example Iddle blueprint, if is not any your way.
Thank you very much.
Good work.

**Comment by aderusha** [comment]
Turns out you're in luck!  There are a couple blueprints that might help you out:

### Dim Screen on Idle

[![Dim Screen on Idle](https://my.home-assistant.io/badges/blueprint_import.svg)](https://my.home-assistant.io/redirect/blueprint_import/?blueprint_url=https%3A%2F%2Fgithub.com%2FHASwitchPlate%2FHASPone%2Fblob%2Fmain%2FHome_Assistant%2Fblueprints%2Fhasp_Dim_Screen_on_Idle.yaml)

Dim the screen backlight after a specified period of inactivity.

### Dim Screen with Sun

[![Dim Screen with Sun](https://my.home-assistant.io/badges/blueprint_import.svg)](https://my.home-assistant.io/redirect/blueprint_import/?blueprint_url=https%3A%2F%2Fgithub.com%2FHASwitchPlate%2FHASPone%2Fblob%2Fmain%2FHome_Assistant%2Fblueprints%2Fhasp_Dim_Screen_with_Sun.yaml)

Dim the HASPone backlight in response to the sun going up and down.

**Comment by proasnet** [comment]
Thank you @aderusha . I know this methods. But hidden buttons is still active and touch to black screen activate scene.
Better will be add switch to blueprint iddle after time, that switch display afzer some time to standby. The first touch turn on display. I can add it to hmi file, but after some your update will be replaced. Is better add this funcion to blueprint.

**Comment by proasnet** [comment]
In my another project, I have tested iddle standby and return by touch:
In all pages/Preinitialize
```
thsp=60
thup=1
```
Then after 60 seconds will display going to standby, and if I touch it, wake up.
This commands may be send via serial.
